README.md

```diff
@@ -111,17 +111,20 @@ You can deploy this project using:
 
 <!-- AUTHORS -->
 
-## 👤Author <a name="author"></a>
+## 👤Authors <a name="author"></a>
 
-> This is a solo project and you can find my contact details below in case you want to reach out:
+> This is a pair programming project and you can find our contact details below in case you want to reach out:
 
 👤 **Ahmed Salah**
 
 - GitHub: [@ahmed](https://github.com/ahmed025566)
 - Twitter: [@AhmedEl18692792](https://twitter.com/AhmedEl18692792)
 - LinkedIn: [Ahmed Salah](https://www.linkedin.com/in/ahmed-salah025566)
 
+👤 **Nurbol Sultanov**
 
+- GitHub: [@everinurmind](https://github.com/everinurmind)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/everinurmind)
 <!-- FUTURE FEATURES -->
 
 ## 🔭 Future Features <a name="future-features"></a>
```

Comment:
- [OPTIONAL] The link to your MIT is broken. It redirects me to an error page. I would advise that you set up a valid LICENSE. Kindly read this [article](https://choosealicense.com/licenses/mit/) and follow the instructions to set up the LICENSE for this project. 👍 😎